Packs/PAN-OS/Playbooks/PAN-OS_-_Extract_IPs_From_Traffic_Logs_To_Sinkhole_README.md

```diff
@@ -0,0 +1,93 @@
+This playbook searches for outgoing traffic to the sinkhole address in PAN-OS. It should be used after a DNS sinkhole was created using the "PAN-OS - Configure DNS Sinkhole" playbook. If a DNS sinkhole was created manually, you should verify in your PAN-OS firewall that there is an address object for the sinkhole FQDN "sinkhole.paloaltonetworks.com", and that there is a rule denying traffic to it from any source. You may specify the name of the deny rule if you know it, or you can let the playbook find the rule automatically.
+
+## Dependencies
+
+This playbook uses the following sub-playbooks, integrations, and scripts.
+
+### Sub-playbooks
+
+This playbook does not use any sub-playbooks.
+
+### Integrations
+
+* Panorama
+
+### Scripts
+
+* Sleep
+* IsIntegrationAvailable
+* Set
+* PrintErrorEntry
+
+### Commands
+
+* pan-os-list-addresses
+* pan-os-list-rules
+* pan-os-query-logs
+
+## Playbook Inputs
+
+---
+
+| **Name** | **Description** | **Default Value** | **Required** |
+| --- | --- | --- | --- |
+| OutputLogsForDetectedIPs | Whether to output full traffic logs for the infected IPs.<br/><br/>If set to True, this will output the logs containing the traffic from the compromised IPs to the sinkhole address. It includes data that can be used for remediation or for deeper investigation by examining what traffic the hosts are creating.<br/><br/>If set to False, data from the logs will not be outputted.<br/><br/>Note: The IP addresses of the systems creating traffic to the sinkhole address will be outputted regardless of playbook input configurations. | False | Required |
+| RuleNameToDenyToSinkhole | The name of the rule that will be created  to deny traffic to the sinkhole address. This is required so that traffic logs will reveal the endpoints attempting to connect to their C&amp;C \(which by the end of this playbook will be sinkholed\).<br/>If a rule with the specified name does not exist, one will be created. The new rule will be placed before rules that allow DNS traffic, as recommended by the PAN-OS best practices. |  | Optional |
+| TimeToWaitForTraffic | The time in seconds that the playbook will wait for hosts to generate traffic to the sinkhole address.<br/>Waiting for long periods of time will increase your chances of finding additional infected systems. However, if enough time passed since your DNS sinkhole was first configure, it may not be needed to wait at all, as systems are already generating traffic to the sinkhole address.<br/>By default, the playbook will wait for 5 minutes \(300 seconds\).<br/>If you do not wish to wait at all, do not delete the value of this input. Instead, you can set the value to 0. | 300 | Required |
```

Comment:
```suggestion
| TimeToWaitForTraffic | The time in seconds that the playbook will wait for hosts to generate traffic to the sinkhole address.<br/>Waiting for long periods of time will increase your chances of finding additional infected systems. However, if enough time passed since your DNS sinkhole was first configured, it may not be needed to wait at all, as systems are already generating traffic to the sinkhole address.<br/>By default, the playbook will wait for 5 minutes \(300 seconds\).<br/>If you do not wish to wait at all, do not delete the value of this input. Instead, you can set the value to 0. | 300 | Required |
```